.travis.yml

```diff
@@ -161,7 +155,7 @@ before_script:
   - if [ "$DB" = 'pgsql' ]; then psql -c 'CREATE DATABASE bedita_test;' -U postgres; fi
   - if [ "$DB" = 'pgsql' ] && [ "$DB_POSTGIS" = '1' ]; then psql -c 'CREATE EXTENSION postgis;' -U postgres; fi
 
-script: vendor/bin/phpunit --coverage-clover=clover.xml # Run PHPUnit.
```

Comment:
I'd keep it in the "minimal" stage. A "reference" stage with only one job will not allow other jobs to run in parallel, thus increasing the overall time required for the build to complete.

Also, since coverage varies slightly from database to database, I'd like to have coverage for at least one job on every database vendor — say, all PHP 7.1 test jobs. If not, things like the fact that a proper error is raised when the database does not support geospatial functions would never be covered.